### PR TITLE
Prep to release version 1.4.0 of this plugin.

### PR DESCRIPTION
**Pull request by peterwilsoncc**
### Description of the Change

See #94

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Changelog Entry

n/a

### Credits

Props @peterwilsoncc

### CODE REVIEW

File: CREDITS.md
Comment (faisal-alvi) [inline CREDITS.md:10]: We can now remove Helen from the Maintainers list and add you?

https://github.com/10up/ads-txt/blob/d4a550263995099145b48ed295f170678868b884/CREDITS.md?plain=1#L7

File: CHANGELOG.md
Comment (faisal-alvi) [inline CHANGELOG.md:4]: Not a related change but line 53 seems to be broken because of the opening of the `<code>` tag when viewing in a [rendered blob mode](https://github.com/10up/ads-txt/blob/d4a550263995099145b48ed295f170678868b884/CHANGELOG.md).

https://github.com/10up/ads-txt/blob/d4a550263995099145b48ed295f170678868b884/CHANGELOG.md?plain=1#L53

**Review by faisal-alvi — CHANGES_REQUESTED** [review]
LGTM! just added minor changes.

Also, I don't find the linked issue which should have a release checklist, for example, https://github.com/10up/ads-txt/issues/57 

Am I missing something?

**Comment by peterwilsoncc** [comment]
> Also, I don't find the linked issue which should have a release checklist, for example, #57
> 
> Am I missing something?

You are not.

I've since created #94 for this purpose.

I've pushed your suggested changes and rebased against the develop branch to include `tested up to 6.0`.

**Review by faisal-alvi — DISMISSED** [review]
🚀

**Comment by peterwilsoncc** [comment]
I rebased this to include the actions fixes in #96 (accounting for rename of release branch to `trunk`). This dismissed the review.

@faisal-alvi nothing else has changed, are you able to reapprove?

**Review by faisal-alvi — APPROVED** [review]
Yes @peterwilsoncc 🚀

**Pull request merged**
